docs/whats-new/release-notes/v3_0_0.md

```diff
@@ -23,12 +23,46 @@ A series of Zowe V3 LTS office hours are available which detail all changes intr
 #### New features and enhancements in Zowe installation and packaging
 
 * Introducing _Zowe Server Install Wizard_, a wizard to simplify installation and configuration of Zowe server side components via a UI.
+* Added explanation to possible error message when checking z/OSMF setting. ([#3956](https://github.com/zowe/zowe-install-packaging/pull/3956))
 
 ### Zowe Application Framework
 
 #### Breaking changes in Zowe Application Framework
 
 * Updated Angular to Version 18 from Version 12. Apps built upon Angular, excluding iframe apps, will need to be updated to be compatible with the V3 Desktop.
+* The Zowe YAML attribute `components.gateway.apiml.security.auth.zosmf.serviceId` for the `gateway` component has changed from `zosmf` to `ibmzosmf`. This may impact Zowe Clients. See `example-zowe.yaml` for new component values. For more information on this change, please see the API Mediation Layer's 3.0.0 Breaking Changes.
+* The Zowe YAML parameter `components.gateway.apiml.security.auth.zosmf. jwtAutoconfiguration` for the `gateway` component has changed. The value `auto` is no longer allowed, and you must choose either the default of `jwt` or `ltpa` depending on if your z/OSMF is set up for JWT use as recommended See `example-zowe.yaml` for new component values. For more information on this change, please see the API Mediation Layer's 3.0.0 Breaking Changes.
```

Comment:
There is still APIML content here that does not belong here

Reply:
Has been removed from this section. Please review and approve.